src/io/c9/ace/Editor.js

```diff
@@ -36,6 +36,12 @@ foam.CLASS({
       name: 'config',
```

Comment:
I think you'll get the same effect if you make the property an FObjectProperty. i.e. try adding:
```
  class: 'FObjectProperty',
  of: 'io.c9.ace.Config'
```
to the property and remove the `adapt`

Reply:
Works like a charm, thanks @mcarcaso.